app/services/container_topology_service.rb

```diff
@@ -89,22 +89,26 @@ def entity_display_type(entity)
     end
   end
 
-  def build_entity_data(entity)
+  def entity_id(entity)
     type = entity_type(entity)
-    status = entity_status(entity, type)
-
     if entity.kind_of?(ManageIQ::Providers::ContainerManager)
       id = entity.id.to_s
     else
       id = case type
-           when 'Vm', 'Host'
-             entity.uid_ems
-           else
-             entity.ems_ref
+             when 'Vm', 'Host'
+               entity.uid_ems
+             else
+               entity.ems_ref
```

Comment:
> @blomquisg @abonas can we just add entity_id methods to these classes as aliases to uid_ems, ems_ref and as id.to_s?

I'd rather not take it to the models because this is a very specific logic for topology, and each model has a bunch of different ids used in different use cases (the rails id, uid_ems, ems_ref, etc), so I think it will create more confusion than benefit (plus a much larger change)